src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -152,11 +147,7 @@ public void Should_give_everyone_and_anonymous_access_rights_when_creating_queue
         [Test]
         public void Should_not_add_everyone_and_anonymous_to_already_existing_queues()
         {
-            var testQueueName = "MsmqQueueCreatorTests.NoChangesToExisting";
-
-            DeleteQueueIfPresent(testQueueName);
-
-            var path = MsmqAddress.Parse(testQueueName).PathWithoutPrefix;
+            var path = MsmqAddress.Parse(testQueueNameForReceiving).PathWithoutPrefix;
 
             using (var existingQueue = MessageQueue.Create(path))
```

Comment:
That was part of the original commit, and not part of my changes. Looks like it slipped past the original review then :). I'll be happy to change it.